code/game/gamemodes/hivemind/hivemind.dm

```diff
@@ -1,3 +1,6 @@
+/datum/game_mode
+	var/datum/objective/hivemind/assimilate_common/assimilation_obj //Create the versus objective here since we want a common target for all the antags
```

Comment:
I feel this would be better as a global or static on antagonist datum.

Reply:
done, made into a static
perhaps the same should be done for exchange objectives some other time